test/routing/RouterProgress.test.tsx

```diff
@@ -47,7 +47,7 @@ const ConfirmationPage = (props: { title: string }) => (
   </Page>
 );
 
-describe('Routing - Router Progress', () => {
+describe.skip('Routing - Router Progress', () => {
```

Comment:
Are the button components breaking tests?

Reply:
Technically yes but we have ran into a few testing issues with web-components like updating input values using react-testing-library doesn't actually update the input field under the web-component, and now that we changed to using the `VaButton` web-component the click event doesn't actually "click" the underlying `<button>` and routing test fail since the page doesn't update.